service/core/serverObj/v2ray.go

```diff
@@ -158,6 +160,12 @@ func ParseVmessURL(vmess string) (data *V2Ray, err error) {
 		}
 	} else {
 		err = jsoniter.Unmarshal([]byte(raw), &info)
+		if info.Host == "" {
```

Comment:
could you please put your code block after the error check?

Reply:
ok